dlvm/tools/scripts/start_agent_and_inf_server.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# GPU Agent
+git clone https://github.com/GoogleCloudPlatform/ml-on-gcp.git
+cd dlvm/gcp-gpu-utilization-metrics
+# Install Python dependencies.
+pip install -r ./requirements.txt
+cp ./report_gpu_metrics.py /root/report_gpu_metrics.py
+
+# Generate GPU service.
+cat <<-EOH > /lib/systemd/system/gpu_utilization_agent.service
```

Comment:
why not include a copy of `gpu_utilization_agent.service` instead of creating a new one?

Reply:
Added files, will test it next iteration, good feedback, thanks !